Podfile

```diff
@@ -468,11 +467,20 @@ post_install do |installer|
 
   # Flag Alpha builds for Tracks
   # ============================
+  #
+  # RuboCop tells us that this loop could be combined with the one above,
+  # because both loop on `installer.pods_project.targets`.
+  # But, from a code organzation point of view, it clearer to separate the two operations,
+  # also because of the `next uless` at the start of each loop.
```

Comment:
```suggestion
  # also because of the `next unless` at the start of each loop.
```

⚠️  If you apply those fixes I suggest on `Podfile`, don't forget to re-run `bundle exec pod install` to update the `Podfile.lock` with the updated checksum

Reply:
Great catch! Done via b6b51ebd9a9f354fbf7506a316eb556d07d0f821